verification/verification.py

```
@@ -524,3 +529,11 @@ async def on_verified_join(self, member: discord.Member):
         channel = await member.guild.fetch_channel(int(os.getenv("WELCOME_CHANNEL")))
         if channel is not None:
             await channel.send(f'Welkom {member.mention}!! 🎊.')
+
+    @tasks.loop(seconds=60)
+    async def check_codes(self):
+        await self.cur.execute("DELETE FROM `verification_codes` WHERE `generated_at` <= datetime('now', '-60 minutes')")
+        deleted = self.cur.rowcount
+        await self.con.commit()
+        if deleted > 0:
+            logging.info(f"[{arrow.now()}] Deleting {str(deleted)} expired verification codes")
```

Ruff (0.14.13)

`info()` call on root logger

(LOG015)